src/docs/introduction/getting-started.md

```diff
@@ -29,6 +30,77 @@ you will be provided with a prompt so that you can choose the type of project to
    app           Minimal starter for building a Stencil app or website
 ```
 
+Selecting the 'component' option will prompt you for the name of your project.
+
+```bash
+✔ Pick a starter › component
+? Project name › my-first-stencil-project
+```
+
+Here, we've named our project 'my-first-stencil project'. After hitting `ENTER` to confirm your choices, the CLI will
+scaffold a Stencil project for us in a directory that matches the project name you provided.
+
+Upon successfully creating our project, the CLI will print something similar to the following to the console:
+```bash
+✔ Project name › my-first-stencil-project
+✔ All setup  in 26 ms
+
+  $ npm start
+    Starts the development server.
+
+  $ npm run build
+    Builds your components/app in production mode.
+
+  $ npm test
+    Starts the test runner.
+
+
+  We suggest that you begin by typing:
+
+   $ cd my-first-stencil-project
+   $ npm install
+   $ npm start
+
+  Further reading:
+
+   - https://github.com/ionic-team/stencil-component-starter
+
+  Happy coding! 🎈
+```
+
+The first section of the output describes a few useful commands available during the development process:
+
+- `npm start` starts a local development server. The development server will open a new browser tab containing your 
+project's components. The dev-server uses hot-module reloading to update your components in the browser as you modify
+them for a rapid feedback cycle.
+
+- `npm run build` creates a production-ready version of your components. The components generated in this step are not
+meant to be used in the local development server, but rather within a project that consumes your components.
+
+- `npm test` runs your project's tests. The Stencil CLI has created both end-to-end and unit tests when scaffolding your project.
+
+The second section describes a few commands to required to finish getting your project bootstrapped.
+
+> You will need to complete these steps in order to run any of the above commands
+
+```bash
+    $ cd my-first-stencil-project
+    $ npm install
+    $ npm start
+```
+
+will change your current directory to `my-first-stencil-project` (or whatever you named your project), install your
```

Comment:
I think "This" might be missing

```suggestion
This will change your current directory to `my-first-stencil-project` (or whatever you named your project), install your
```